tests/gold_tests/remap/remap_acl.test.py

```diff
@@ -296,3 +309,12 @@ def _configure_client(self, tr: 'TestRun') -> None:
     acl_configuration='@action=allow @in_ip=3.4.5.6 @method=GET @method=POST',
     named_acls=[],
     expected_responses=[200, 403, 403, 403, 403])
+
+test_named_acl_deny = Test_remap_acl(
+    "Verify we can deny all but allow in a named acl.",
+    replay_file='get_head_post.replay.yaml',
+    ip_allow_content=IP_ALLOW_MINIMUM,
+    deactivate_ip_allow=False,
+    acl_configuration='',
+    named_acls=[('deny', '@action=deny @method=HEAD @method=POST')],
+    expected_responses=[200, 403, 403])
```

Comment:
@bneradt It's not clear to me what adding a `PUSH` would prove that isn't already proven by the existing methods?

Reply:
Ah, right. With the other IP_ALLOW config used in this test, it just allows GET. With that config, testing PUSH would verify that the ip_allow rule isn't being used if the PUSH were _accepted_ with a 200 since the ip_allow rule would otherwise reject it. The point is, we want to test things in such a way to explicitly verify that only one of these rules (the one named ACL you're adding) is being used.

So here's my suggestion:

* Let's add a PUSH transaction with the expectation of a 200 response.
* Let's use the other ip_allow.yaml config. The ones the other tests use. Note that this ip_allow configuration would reject a PUSH with a 403 rather than accept it.
* Let's verify that the PUSH gets a 200 response since the activated named ACL you've defined should not reject it.

That will verify that the ip_allow rule is not being used.

The reason this is significant to demonstrate is that historically there was a complex ACL relationship with `deny`. `deny` used to _only_ deny the specified methods, with non-matching methods falling through to other remap ACL rules or ip_allow rules. This made thinking through `deny` ACL rules difficult. This got fixed so that `deny` denies the specified methods and accepts all others, without other rules being processed. We want to demonstrate this is still the case.

Thank you for carefully thinking through this.